summingbird-storm/src/main/scala/com/twitter/summingbird/storm/FlatMapBoltProvider.scala

```diff
@@ -24,9 +24,12 @@ import com.twitter.summingbird.storm.option.AnchorTuples
 import com.twitter.summingbird.planner._
 import com.twitter.summingbird.online.executor
 import com.twitter.summingbird.online.FlatMapOperation
+import com.twitter.summingbird.storm.builder.Topology
 import com.twitter.summingbird.storm.planner._
 import org.slf4j.LoggerFactory
 
+import scala.collection.{ Map => CMap }
```

Comment:
We use `CMap` in other places in SB. I don't know why and I like `ScalaMap` more, but I guess it makes sense to leave it as is for now.